doc/.wordlist.txt

```diff
@@ -7,6 +7,7 @@ APIs
 balancer
 Charmhub
 CLI
+devLXD
```

Comment:
Shouldn't need this. I think we should be using DevLXD everywhere (or `devlxd` if necessary).